arch/arm64/src/common/arm64_gicv3.c

```diff
@@ -577,8 +613,8 @@ static void arm64_gic_init(void)
   int       err;
 
   cpu               = this_cpu();
-  gic_rdists[cpu]   = CONFIG_GICR_BASE +
-                     MPIDR_TO_CORE(GET_MPIDR()) * 0x20000;
+
+  gic_rdists[cpu]   = g_arm64_cpu_info[cpu].gicr_base;
```

Comment:
can we add arm64_gic_get_gicr_base or change 0x20000 to a macro defined in chip.h?